drivers/eaton-ats16-mib.c

```diff
@@ -259,8 +259,8 @@ static snmp_info_t eaton_ats16_mib[] = {
 };
 
 /* Note: keep the legacy definition intact, to avoid breaking compatibility */
+/*mib2nut_info_t  eaton_ats16 = { "eaton_ats16", EATON_ATS16_MIB_VERSION, NULL, EATON_ATS16_MODEL, eaton_ats16_mib, EATON_ATS16_SYSOID_GEN1 }; */
 mib2nut_info_t	eaton_ats16 = { "eaton_ats16", EATON_ATS16_MIB_VERSION, NULL, EATON_ATS16_MODEL, eaton_ats16_mib, EATON_ATS16_SYSOID_GEN1 };
-/* Newer Network-M2 communication cards, with a fixed sysOID */
```

Comment:
Like in comments for upstream NUT PR sibling, what does this change beside removing a (possibly useful?) comment, and adding a commented line whose payload seems identical to the line under it?